cypress/e2e/emergencyProposals/proposals-emergency-batch-voting.cy.ts

```diff
@@ -1,78 +0,0 @@
-it.skip("Proposals", () => {
```

Comment:
why is it deleted? it was skipped to start from

Reply:
It is a very old test, I would like to completely re-write it if needed in the future, more so if we are changing the layout of the whole application